Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -1882,6 +1882,18 @@ HTTPServer server({
     EXPECT_FALSE(canLoadURLInIFrame(@"/always_blocked"));
 }
 
+TEST(SiteIsolation, LoadRequestOnOpenerWebView)
+{
+    HTTPServer server({
+        { "/example"_s, { "<script>w = window.open('https://webkit.org/webkit')</script>"_s } },
+        { "/webkit"_s, { ""_s } }
+    }, HTTPServer::Protocol::HttpsProxy);
+    auto [opener, opened] = openerAndOpenedViews(server);
+    [opener.webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://apple.com/webkit"]]];
+    [opener.navigationDelegate waitForDidFinishNavigation];
+    checkFrameTreesInProcesses(opener.webView.get(), { { "https://apple.com"_s } });
```

Comment:
Let's check the opened webView's frame trees too.

Reply:
What I was trying to say is that it is worth having two calls to checkFrameTreesInProcesses, one for the opener and one for the opened.

Reply:
Oh, will do that too.